sdk/kiosk/src/query/transfer-policy.ts

```diff
@@ -44,15 +46,11 @@ export async function queryTransferPolicy(
         );
       }
 
-      let parsed = bcs.de(
-        '0x2::transfer_policy::TransferPolicy',
-        policy.bcs.bcsBytes,
-        'base64',
-      );
+      let parsed = bcs.de(TRANSFER_POLICY_TYPE, policy.bcs.bcsBytes, 'base64');
 
       return {
```

Comment:
I think that we could help provide initialSharedVersion for shared transfer policy. That could help build arguments for the purchase flow.